docker/docker-compose.suite-ci.yml

```diff
@@ -5,9 +5,10 @@ services:
     environment:
       - SDL_VIDEODRIVER=dummy
       - XDG_RUNTIME_DIR=/var/tmp
+    network_mode: bridge # this makes docker reuse existing networks
 
   test-run:
-    image: cypress/included:4.7.0
+    image: cypress/included:5.0.0
```

Comment:
I have noticed that latest cypress is 6.x. Do we upgrade to 5.x just to have it gradually or we need to stick with 5 for some reason?

Reply:
Just to make this PR easier. 6.1.0 comes in another one https://github.com/trezor/trezor-suite/pull/3159

Reply:
You are the best :rocket: